build/sources.props

```diff
@@ -10,6 +10,9 @@
 
     <RestoreSources Condition=" '$(DotNetBuildOffline)' != 'true' ">
       $(RestoreSources);
+      https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json;
+      https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json;
+      https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json;
```

Comment:
Doesn't this repo also publish to https://dotnetfeed.blob.core.windows.net? I'm not seeing a change there.

Reply:
Can you clarify? Not sure I understand....

![image](https://user-images.githubusercontent.com/2696087/53276814-b11da000-36b5-11e9-8b54-63d7b7950575.png)

Reply:
The "Change publish feed to workaround 'Waiting to obtain an exclusive lock on the feed.'" PRs in other repos also updated azure-pipelines.yml. I was expecting a similar change in .azure/pipelines/ci.yml Now, I suspect that expectation was off base.

Reply:
Oh, I understand. Yes, I'll add the appropriate change for this repo. It's in a different spot, not azure-pipelines.yml. Good catch.